ansible/group_vars/all/defaults.yaml

```diff
@@ -5,17 +5,17 @@ python_path: ""
 # If it's not there, the kubernetes_full_version will have "None" for a version number.
 #
 # IMPORTANT When you update kubernetes_version, also update crictl_version.
-kubernetes_version: "1.29.6"
+kubernetes_version: "1.30.5"
 kubernetes_major_minor: "{{ (kubernetes_version.split('.') | map('trim'))[:2] | join('.') }}"
 kubernetes_semver: "v{{ kubernetes_version }}"
 
-kubernetes_cni_version: "1.3.0"
+kubernetes_cni_version: "1.5.1"
 
 # The crictl CLI is released as part of the http://sigs.k8s.io/cri-tools project.
 # The project release closely follows the Kubernetes release cycle, and uses a
 # nearly identical version scheme.
 # IMPORTANT When you update crictl_version, also update crictl_sha256.
-crictl_version: "{{ kubernetes_major_minor }}.0"
+crictl_version: "{{ kubernetes_major_minor }}.1"
```

Comment:
Yes I think you're right here it should be .1
When I run the install from the upstream repo, I see
```
 cri-tools aarch64 1.30.1-150500.1.1
 ```
In that case we need to also update https://github.com/mesosphere/dkp-release/blob/175d9d167cd5015ed26f430b29463eef1ea65907/k8s-build/package_rpms.sh#L44-L53

Reply:
@dkoshkin PR: https://github.com/mesosphere/dkp-release/pull/145

Reply:
We want to make another change in https://github.com/mesosphere/konvoy-image-builder/blob/73172e0a5abc96ac936da338b3b94b6dad3ffd97/Dockerfile.devkit#L90 

```
export CRICTL_TOOLS_VERSION="$(echo ${KUBERNETES_VERSION} | cut -d. -f1-2).1"
```

Reply:
done

Reply:
One more for the FIPS packages https://github.com/mesosphere/konvoy-image-builder/blob/c3c49c5467207b79f30b30c2d26d8b807bce65e4/Dockerfile.devkit#L103

Reply:
For the Ubuntu test, 
It should be:
```
Get:1 https://prod-cdn.packages.k8s.io/repositories/isv:/kubernetes:/core:/stable:/v1.30/deb  cri-tools 1.30.1-1.1
```

Please update it here and correct the comment.
https://github.com/mesosphere/konvoy-image-builder/blob/73172e0a5abc96ac936da338b3b94b6dad3ffd97/cmd/konvoy-image-wrapper/cmd/create-package-bundle.go#L295-L296

Reply:
done